alacritty_terminal/tests/ref/colored_underline/alacritty.recording

```diff
@@ -0,0 +1,3 @@
+[1m[7m%[27m[1m[0m                                                                               
 

[0m[27m[24m[Jpohjala% [K[?2004h

[0m[27m[24m[Jpohjala% [7mecho "\u1b[58;5;2;38;5;3mNO\u1b[4mUND\u1b[58;5;6mERLI\u1b[59mNE\u1b[0mD"[27m[72D[27me[27mc[27mh[27mo[27m [27m"[27m\[27mu[27m1[27mb[27m[[27m5[27m8[27m;[27m5[27m;[27m2[27m;[27m3[27m8[27m;[27m5[27m;[27m3[27mm[27mN[27mO[27m\[27mu[27m1[27mb[27m[[27m4[27mm[27mU[27mN[27mD[27m\[27mu[27m1[27mb[27m[[27m5[27m8[27m;[27m5[27m;[27m6[27mm[27mE[27mR[27mL[27mI[27m\[27mu[27m1[27mb[27m[[27m5[27m9[27mm[27mN[27mE[27m\[27mu[27m1[27mb[27m[[27m0[27mm[27mD[27m"[?2004l


```

Comment:
This ref test is rather big and I see no reason why it couldn't be combined with the underline ref test. Would you mind re-recording it to include the tests of both so the size is a bit smaller? Minimizing window size will also help with reducing the output grid.json a bit.